tests/test_salesforce_gen.py

```diff
@@ -11,3 +14,12 @@ def test_content_version(self, generated_rows):
         rawdata = b64decode(b64data)
         assert rawdata.startswith(b"%PDF-1.3")
         assert b"Helvetica" in rawdata
+
+
+class TestSalesforceConnection:
+    def test_bad_kwargs(self):
+        sfc = SalesforceConnection(None)
+        with pytest.raises(exc.DataGenError, match="Unknown argument"):
```

Comment:
Is the unknown argument `xyzzy` or is it the `blah` field? It would be nice if the error message included the name of the unknown argument.